docker-compose.yml

```diff
@@ -46,7 +46,7 @@ services:
       - ./backend:/app
     environment:
       - MONGODB_HOST=rosalution-db
-      - ROSALUTION_KEY
+    entrypoint: ['/bin/sh', '-c', './etc/entrypoint-init.sh']
```

Comment:
A little obfuscated as to where the key is coming from/how it fits in, but this was discussed and is intended. 

Everything is working with the changes!